homeassistant/components/kostal_plenticore/sensor.py

```diff
@@ -179,11 +188,21 @@ def device_class(self) -> str | None:
         """Return the class of this device, from component DEVICE_CLASSES."""
         return self._sensor_data.get(ATTR_DEVICE_CLASS)
 
+    @property
+    def state_class(self) -> str | None:
+        """Return the class of the state of this device, from component STATE_CLASSES."""
+        return self._sensor_data.get(ATTR_STATE_CLASS)
+
     @property
     def entity_registry_enabled_default(self) -> bool:
         """Return if the entity should be enabled when first added to the entity registry."""
         return self._sensor_data.get(ATTR_ENABLED_DEFAULT, False)
 
+    @property
+    def last_reset(self) -> str | None:
```

Comment:
This will return a datetime.
```suggestion
    def last_reset(self) -> datetime | None:
```

Reply:
It produced linting errors on the last run, that‘s why I changed it

Reply:
Ig we just need to import `datetime`